site/src/sitecomponents/BadgeGrid/index.js

```diff
@@ -24,49 +24,15 @@ const Footer = () => {
-        </div>
-        <div>
-          <img src="assets/badges/meshery/meshery.png" alt="Layer5 badges" />
-          <span>Meshery</span> <p>This badge isawarded to the community members who make consistent and impactful contributions to the Meshery project. Community members who earn this badge occasionally become a project maintainer.</p>
-        </div>
-        <div>
-          <img src="assets/badges/patterns/patterns.png" alt="Layer5 badges" />
-          <span>Patterns</span> <p>This badge is awarded to the community members who make consistent and impactful contributions to the <a href="https://service-mesh-patterns.github.io/service-mesh-patterns">Service Mesh Patterns</a> project in recognition and appreciation of their efforts.</p>
-        </div>
-        <div>
-          <img src="assets/badges/landscape/landscape.png" alt="Layer5 badges" />
-          <span>Landscape</span> <p>This badge is awarded to the community members who make consistent and impactful contributions to the layer5.io website.</p>
-        </div>
-        <div>
-          <img src="assets/badges/nighthawk/nighthawk.png" alt="Layer5 badges" />
-          <span>Nighthawk</span> <p>This badge is awarded to the community members who make consistent and impactful contributions to the NightHawk project in recognition and appreciation of their efforts. Community members who earn this badge occasionally become a project maintainer.</p>
-        </div>
-        <div>
-          <img src="assets/badges/meshery-catalog/meshery-catalog.png" alt="Layer5 badges" />
-          <span>Meshery Catalog</span> <p>This badge is awarded to the community members who make consistent and impactful contributions to the <a href="https://meshery.io/catalog">Meshery Catalog</a> of Meshery project in recognition and appreciation of their efforts.</p>
-        </div>
-        <div>
-          <img src="assets/badges/docker-extension/docker-extension.png" alt="Layer5 badges" />
-          <span>Docker Extension</span> <p>This badge is awarded to the community members who make consistent and impactful contributions to the Docker Extension of meshery project in recognition and appreciation of their efforts.</p>
-        </div>
-        <div>
-          <img src="assets/badges/meshery-docs/meshery-docs.png" alt="Layer5 badges" /> <span>Meshery Docs</span> <p>This badge is awarded to the community members who make consistent and impactful contributions to the <a href="https://docs.meshery.io">Meshery Docs</a> in recognition and appreciation of their efforts.</p>
-        </div>
+        {jsonData.projectBadges.map((badge) => {
+          return (
+            <div key={badge.slug}>
+              <img src={badge.imageSource} alt="Layer5 badges" />
+              <span>{badge.title}</span>
+              <p>{badge.description}</p>
```

Comment:
But if you render the description directly, then the anchor tags will not be applied as they consider them as a string.

Reply:
Think of some new approach to overcome this issue. If you don't get any idea, then feel free to ask me for help!

Reply:
Since it's taking you long to solve this, Go through this [reference](https://stackoverflow.com/a/45058774)

Reply:
yeah got it.